client/templates/userprofile/settings.js

```diff
@@ -82,26 +110,40 @@ Template.profileSettings.events({
         {
             Session.set("emailerror", "Your Email is empty!");
         }
+        setTimeout(function() {
+         $('.username_errors').text("");
+        }, 5000);
 
     },
 
     'click #saveChanges': function(event, template) {
-        var realname = template.find("#realName").value;
+        // realname is for later use
+        //var realname = template.find("#realName").value;
+        
         var username = template.find("#userName").value;
         var gender = template.find("#gender option:selected").value;
         var language = template.find("#language option:selected").value;
         var biography = template.find("#biography").value;
         var email = template.find("#email").value;
-
+        if (biography.length > 300)
+        {
+            $("#bio-error").text("You may not have more than 300 Characters in your Biography");
```

Comment:
These kinds of field validation should be done with the AutoForm/Simple Schema packages.